src/components/1j1s/header-1j1s.vue

```diff
@@ -24,8 +24,13 @@
                 </button>
               </div>
             </div>
-            <div class="fr-header__service">
-              <a :href="parameters.homeTo" :title="parameters.homeTitle">
+            <div class="fr-header__service aj-header">
+              <a
+                :href="parameters.homeTo"
+                :title="parameters.homeTitle"
+                :target="openInNewTab ? '_blank' : '_self'"
```

Comment:
openInNewTab n'est pas déclaré dans ce composant, cependant on peut faire plus simple : 

```suggestion
                :target="store.inIframe ? '_blank' : '_self'"
```

Reply:
Ok comme le store est deja utilisé je vais faire ça. en revanche, je pense qu'on pourrait a terme : 
- supprimer la dependance au store dans ce composant pour le rendre juste fonctional
- déclarer le props `collapse` (j'avoue j'ai utilisé l'existant qui n'est pas fou même si ça marche). 

Je simplifie la PR en faisant ce que tu dis